CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.4.1 (2019-02-21)
+
+- Fixed radio and checkbox labels to align with chexboxes and radio buttons using a top baseline. [#50](https://github.com/blackbaud/skyux-theme/pull/50)
+- Fixed radio and checkbox labels to wrap contents when overflowing a parent element. [#50](https://github.com/blackbaud/skyux-theme/pull/50)
```

Comment:
Does meaning change if it's worded to be:
"Fixed radio and checkbox labels to wrap contents when a parent element overflows." 
it meaning doesn't change, my preference is "when a parent element overflows."

Reply:
Ya. Technically it would be the label that is overflowing the parent, not the parent overflowing. 

I like "when overflowing its parent element" but wasn't sure if that was proper style for a changelog.....

Reply:
In that case I think what you had originally is good 👍

Reply:
Sounds good.